app/src/main/res/values/dimens.xml

```diff
@@ -5,7 +5,16 @@
     <dimen name="activity_vertical_margin">16dp</dimen>
     <dimen name="fab_margin">16dp</dimen>
     <dimen name="name_font_size">16sp</dimen>
+
+    <dimen name="pet_favorites_profile_pic_width">96dp</dimen>
+    <dimen name="pet_favorites_profile_pic_height">96dp</dimen>
+
     <dimen name="pet_details_image_height">340dp</dimen>
     <dimen name="pet_detail_title_bottom_padding">10dip</dimen>
 
+    <!-- Swipe Card Dimensions -->
+    <dimen name="pet_swipecard_image_height">400dp</dimen>
+    <dimen name="pet_swipecard_height">450dp</dimen>
+    <dimen name="pet_swipecard_frame_height">470dp</dimen>
+    
```

Comment:
the framing is a little hacky in that it requries explicit heights for the image, the card and the frame. Would prefer it to be more flexible but this should do for now.

When Scott starts adding typography to this layout we'll want to change all three of these dimensions together in order to create more space on the card for rows of text.